### ISSUE TITLE
Compiling without Docker?

### ISSUE THREAD

**Issue by Tyberkid967** [fgsfdsfgs/sm64-port#35]
I just wanted to ask if you can compile the dos branch without docker.
(If it helps i am running Windows 10)

**Comment by fgsfdsfgs** [comment]
Yes. You need to install MSYS2, get DJGPP from [here](https://github.com/andrewwutw/build-djgpp/releases/tag/v3.1), install it, add it to `PATH` by doing something like
```
export PATH="/path/to/where/you/put/djgpp/bin:$PATH"
export GCC_EXEC_PREFIX=/path/to/where/you/put/djgpp/lib/gcc/
export DJDIR=/path/to/where/you/put/djgpp/i586-pc-msdosdjgpp
```
in the MINGW32 terminal (or by adding it to `msys2/home/<username>/.bashrc`), then you just clone the repo and run `make`.
Could also work in WSL, in which case you'd need to get DJGPP for Linux from the link in the Dockerfile.
